File: sentimental-mario-less/mario.py
# Program that prints Mario pyramid

# Prompting user for input between 0 and 9
while True:
    try:
        height = int(input("Height: "))
    except ValueError:
        continue
    if 1 <= height <= 8:
        break

# Drawing pyramid
if height == 1:
    print("#")
else:
    # Loop for rows
    for i in range(1, height + 1):
        # Loop for spaces
        for j in range(height - i):
            print(" ", end="")
        # Loop for hashes
        for k in range(i):
            print("#", end="")
        print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompting user for input
    int number = get_int("Height: ");

    // prompting unless user inputs numbers between 1 and 8 (inclusive)
    if (number < 1 || number > 8)
    {
        main();
    }
    else
    {
        // loop for new line
        for (int i = 0; i < number; i++)
        {
            // loop for spaces
            for (int k = 1; k < number - i; k++)
            {
                printf(" ");
            }
            // loop for hashes
            for (int j = 0; j < i + 1; j++)
            {
                printf("#");
            }
            printf("\n");
        }
    }
}
